.circleci/config.yml

```diff
@@ -149,6 +149,23 @@ jobs:
           paths:
             - ./vendor
       - run: ./tools/ci/ethereum_test parity
+  lint:
+    working_directory: /go/src/github.com/smartcontractkit/chainlink
+    docker:
+      - image: smartcontract/builder:1.0.17
+    steps:
+      - checkout
+      - run: echo $CACHE_VERSION > cache.version
+      - restore_cache:
+          name: Restore Yarn Package Cache
+          key: v{{ checksum "cache.version" }}-yarn-vendor-{{ checksum "yarn.lock" }}
+      - run: yarn install
+      - save_cache:
+          name: Save Yarn Package Cache
+          key: v{{ checksum "cache.version" }}-yarn-vendor-{{ checksum "yarn.lock" }}
+          paths:
+            - /usr/local/share/.cache/yarn
+      - run: ./tools/ci/lint
```

Comment:
1. In an effort to reduce the number of build works, can we include this in another build? Like `gui`?
1. Or perhaps have each build, gui and solidity, be responsible for invoking their own `link` test?

Reply:
@dimroc I feel like the time to spin up and lint multiple environments would be > than 1 lint job. It takes ~40s before we even start linting and ~7s to lint all the files.

Reply:
I think we are in agreement in the comments, in that spinning up a new build is time consuming, while simply running lint is cheap. The implementation however spins up a new build, which is what I'm trying to avoid. Let's sync up offline.

Reply:
Gotcha. I misunderstood it as a new one for each group